modules/manifold/actions.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/esm/actions');
```

Comment:
These files allows for `import {something} from '@mlvis/manifold/some-subdirectory';`

Reply:
Not sure if this is the best practice to reference a specific build since there are separate builds for es6, es5, esm for different compatibility layers. This require breaks that expectation.

Reply:
Current plan is to only support esm build based on recommendation from @ibgreen for enabling tree shaking. Currently, `ocular-build` cannot be configured to disable other builds. Supporting other builds will happen in the future.

Reply:
Cool, let's make sure this is captured in the PR's description.